contribution.md

```diff
@@ -0,0 +1,7 @@
+## Project Contributions
```

Comment:
Provide description of project according to the homework instructions, put in correct folder (like `projects/python`)